KalturaWowzaServer/src/main/java/com/kaltura/media/server/wowza/listeners/LiveStreamEntry.java

```diff
@@ -220,7 +220,13 @@ public void onPublish(IMediaStream stream, String streamName, boolean isRecord,
 
 					String streamSuffix = matcher.group(2);
 					KalturaLiveAsset liveAsset = liveStreamManager.getLiveAsset(entry.id, streamSuffix);
-					if (liveAsset != null) {
+					if (liveAsset == null) {
```

Comment:
A reference (count) is still added for this stream - is it removed on rejectConnection?